src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -728,6 +734,76 @@ private void createGitHubRepository(Channel channel, User sender, String name, S
         }
     }
 
+    /**
+     * Makes GitHub team visible.
+     *
+     * @param team
+     *      team to make visible
+     */
+    private boolean makeGitHubTeamVisible(Channel channel, User sender, String team) {
+        boolean result = false;
+        if (!isSenderAuthorized(channel, sender)) {
+            insufficientPermissionError(channel);
+            return false;
+        }
+        OutputChannel out = channel.send();
+        try {
+            GitHub github = GitHub.connect();
+            GHOrganization o = github.getOrganization(IrcBotConfig.GITHUB_ORGANIZATION);
+
+            final GHTeam ghTeam = o.getTeamByName(team);
+            if (ghTeam == null) {
+                out.message("No team for " + team);
+                return false;
+            }
+
+            ghTeam.setPrivacy(GHTeam.Privacy.CLOSED);
+
+            String successMsg = "Made GitHub team " + team + " visible";
```

Comment:
Any reason to have the as a separate declaration? It doesn't conform to the other `out.message` calls above and below.

Reply:
It conforms to where it was copy pasted from
```
private boolean addGitHubCommitter(Channel channel, User sender, String collaborator, String justForThisRepo) {
```

Happy to change it, but the code is adapted from similar code